cmd/integration_test/cmd/cleanupaws.go

```diff
@@ -49,6 +50,7 @@ func init() {
 	cleanUpAwsCmd.Flags().StringP(storageBucketFlagName, "s", "", "Name of s3 bucket used for e2e testing")
 	cleanUpAwsCmd.Flags().StringP(maxAgeFlagName, "a", "0", "Instance age in seconds after which it should be deleted")
 	cleanUpAwsCmd.Flags().StringP(tagFlagName, "t", "", "EC2 instance tag")
+	cleanUpAwsCmd.Flags().StringP(buildIdFlagName, "b", "", "Build ID of CodeBuild build; will clean up instances associated with the build")
```

Comment:
Following the other comment, I would remove these two tag flags and just take `--tags`